web-api/src/models.rs

```diff
@@ -313,6 +313,9 @@ pub(crate) struct IngestedDocument {
     /// Snippet used to calculate embeddings for a document.
     pub(crate) snippet: DocumentSnippet,
 
+    /// Indicates weather the embedding was created using a summarizer.
+    pub(crate) was_summarized: bool,
```

Comment:
:art: naming convention
```suggestion
    pub(crate) is_summarized: bool,
```